src/components/App/App.js

```diff
@@ -63,15 +63,15 @@ class App extends Component {
           </section>
         </Route>
         <Route path="/mysongs">
-          <Navigation dynamic="off-home-nav" />
+            <Navigation dynamic="off-home-nav" />
           { !!this.state.mySongs.length &&
           <MySongLibrary
           songs={ this.state.mySongs }
           mySongs={this.state.mySongs }
           handleSong={ this.removeSong }
           buttonIcon={ [<MdRemoveCircle className="handle-song-icon"/>] }
           /> }
-          { this.state.error && <h2>{this.state.error}</h2> }
+          { this.state.error && <p>{this.state.error}</p> }
```

Comment:
Didn't know that WAVE is so particular with headings, will keep this in mind for the future